README.md

```diff
@@ -1,28 +1,74 @@
-#ESP8266 USB Software Driver
-
-This project only works with USB Low Speed.  Additionally, you cannot use SDKs newere than 1.5.4.  All ESP SDK 2.0 SDKs are incompatible.
-
-Forum for discussion of development: http://www.esp8266.com/espusb
+# ESP8266 USB Software Driver
 
 I wanted to have USB on the ESP8266, and a while ago I saw on the ESP32 flier, it read: "Rich Peripherals" ... "Sorry, no USB!" I thought to myself, that is ridiculous.  Of course there's USB.
 
 So, it was born.  This is a software USB stack running on the ESP8266/ESP8285.  It requires only one external resistor, between D- and 3.3V.
 
-Currently, the chip expects D- on GPIO 4 and D+ on GPIO 5 - but any GPIO pair may be used.  It is important to note that both D- and D+ MUST be adjacent.
+## Limitations
+
+- You cannot use SDKs newer than 1.5.X. (As of dec-02-2016, 1.5.4 is the latest SDK that has been verified to be compatible.)
+- All ESP SDK 2.0 SDKs are incompatible.
+- By default, the chip expects D- on GPIO 4 and D+ on GPIO 5 - but any GPIO pair may be used.  It is important to note that both D- and D+ MUST be adjacent.
+- This project only operates with low-speed USB (1.5 MBit/s)
+  ideal for making peripherals, not for fake network devices and usb-serial bridges.  
```

Comment:
USB-Serial bridges means emulating a Serial Port? I think that's possible since there was other projects in AVR that emulated Serial comm over vUSB. I'm currently attempting to try this but because newer kernels don't recognize low-speed usb cdc, there's some errors. [https://github.com/cnlohr/espusb/issues/13]